src/NzbDrone.Core/ImportLists/Plex/PlexSettingsBase.cs

```diff
@@ -0,0 +1,88 @@
+using System;
+using System.Text.RegularExpressions;
+using FluentValidation;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Annotations;
+using NzbDrone.Core.ThingiProvider;
+using NzbDrone.Core.Validation;
+
+namespace NzbDrone.Core.ImportLists.Plex
+{
+    public class PlexSettingsBaseValidator<TSettings> : AbstractValidator<TSettings>
+    where TSettings : PlexSettingsBase<TSettings>
+    {
+        public PlexSettingsBaseValidator()
+        {
+            RuleFor(c => c.AccessToken).NotEmpty()
+                           .OverridePropertyName("SignIn")
+                           .WithMessage("Must authenticate with Plex");
+
+            // Loose validation @TODO
+            RuleFor(c => c.Rating)
+                .Matches(@"^\d+\-\d+$", RegexOptions.IgnoreCase)
+                .When(c => c.Rating.IsNotNullOrWhiteSpace())
+                .WithMessage("Not a valid rating");
+
+            // Any valid certification
+            RuleFor(c => c.Certification)
+                .Matches(@"^\bNR\b|\bG\b|\bPG\b|\bPG\-13\b|\bR\b|\bNC\-17\b$", RegexOptions.IgnoreCase)
+                .When(c => c.Certification.IsNotNullOrWhiteSpace())
+                .WithMessage("Not a valid cerification");
+
+            // Loose validation @TODO
+            RuleFor(c => c.Years)
+                .Matches(@"^\d+(\-\d+)?$", RegexOptions.IgnoreCase)
+                .When(c => c.Years.IsNotNullOrWhiteSpace())
+                .WithMessage("Not a valid year or range of years");
+
+            // Limit not smaller than 1 and not larger than 100
+            RuleFor(c => c.Limit)
+                .GreaterThan(0)
+                .WithMessage("Must be integer greater than 0");
+        }
+    }
+
+    public class PlexSettingsBase<TSettings> : IProviderConfig
+        where TSettings : PlexSettingsBase<TSettings>
+    {
+        protected virtual AbstractValidator<TSettings> Validator => new PlexSettingsBaseValidator<TSettings>();
+
+        public PlexSettingsBase()
```

Comment:
Out of scope for this PR. But it might make sense to make a common import list settings base for all these filters that seem to be somewhat common in all lists.